scripts/check_compliance.py

```diff
@@ -103,6 +99,10 @@ class ComplianceTest:
     path_hint:
       The path the test runs itself in. This is just informative and used in
       the message that gets printed when running the test.
+
+      The magic string "<repo>" is replaced with the top-level repository
+      directory. This avoids running 'git' before _main(), which would prevent
+      errors from it from being reported to GitHub.
```

Comment:
Can you rephrase a bit to briefly mention that `_main()` is the one reporting to github? I forgot that and was really struggling.

Reply:
Rephrased and added some more details:

```
The magic string "<git-top>" refers to the top-level repository directory. This
avoids running 'git' to find the top-level directory before main() runs (class
variable assignments run when the 'class ...' statement runs). That avoids
swallowing errors, because main() reports them to GitHub.
```

Is that okay?